docs/product-getting-started-guide/src/main/asciidoc/task_data-model.adoc

```diff
@@ -1,66 +1,157 @@
 [id='_defining_a_data_model']
-= Defining a Data Model
+= Creating the traffic violation data model
 
-The following chapter instructs you how to create a data model in {PRODUCT}.
+The following chapter instructs you how to create a traffic violation data model in {PRODUCT}. Data models are models of data objects. A data object is a custom complex data type (for example, a Violation data object containing data fields Violation Date, Fine Amount, and Speed Limit).
+
+Data models are saved in data models definitions stored in your Project. {PRODUCT} provides the Data Modeler, a custom graphical editor, for defining data objects. *<MH> Ivo, I think that this paragraph might be a bit too low-level for the purpose of this tutorial. If you agree, I will remove it. Please advise</MH>*
+
+The following illustration shows the traffic violation data model that you will create in this chapter. The data model consists of data objects containing data fields. *<MH>Ivo, please let me know ASAP if the following image is a good base to send off to the graphics team.</MH>*
+
+.Traffic violation data model
+image:data-model.png[]
 
 [float]
 == Prerequisites
 
-Imported starter project. For further information, see <<_importing_a_getting_started_business_project>>.
-
-== Creating a Data Model
+You have created the Driver department traffic violations project.
 
-To create a data model:
+== Creating the Violation Data Object
 
-. Log in to Business Central. Click *Menu* -> *Design* -> *Projects*, then *process-app-start*.
```

Comment:
Be sure to replace all instances of "Business Central", and even any instances of hard-coded "Decision Central", with {CENTRAL}.

Reply:
Searched and destroyed all instances. Thanks!